extension.js

```diff
@@ -102,6 +102,30 @@ function applyPermissions(path, { resource, acls }) {
 			}
 			return super.allowCreate(user);
 		}
+
+		/**
+		 * Check if the user is allowed to update the topic (resource)
+		 * @param user
+		 * @param query
+		 * @param context
+		 * @return {*|boolean}
+		 */
+		allowUpdate(user, query, context) {
+			let id = this.getId();
```

Comment:
or dedupe the whole block with `return this.allow(user, query, context)`... (however you want to do it, code duplication does present a maintenance hazard (keeping them in sync)